pyqt-apps/siriushla/widgets/label.py

```diff
@@ -104,6 +105,13 @@ def value_changed(self, new_value):
             self.setText(time)
             return
 
+        # If it is a version string, replace multiple whitespaces with a single one
+        if self._display_format_type == self.DisplayFormat.Version:
+            version = new_value[:16] + " " + new_value[16:]
```

Comment:
isn't this too particular? Why splitting at the 16th position?

Reply:
As per BSMP specifications regarding UDCs, version strings shall be exactly 16 characters long, always. However, we can rename this `DisplayFormat` to something like "UDCVersion" as this is the standard for ARM/DSP firmware on those.

Reply:
I think it is a good ideia to change the name of the 'DisplayFormat' option then. This widget is used in almost all of our windows and it is not a good practice to let its interface dubious.

Reply:
`Version` was changed to `BSMPUDCVersion`. It is indeed a better naming option